js/validation/index.js

```diff
@@ -65,12 +65,12 @@ function validateDocument(doc) {
 }
 
 
-async function parseAndValidateCodemeta(showPopup) {
+function parseAndValidateCodemeta(showPopup) {
```

Comment:
Go back to former version and do not mix json ld procesing with validation before #29 .